pixi.toml

```diff
@@ -11,10 +11,11 @@ platforms = ["win-64", "linux-64"]
 
 [tasks]
 # Installation
-install-julia = "juliaup add 1.10.5 && juliaup override unset && juliaup override set 1.10.5"
+install-julia = "juliaup add 1.10.7 && juliaup override set 1.10.7"
```

Comment:
This can be simplified due to https://github.com/JuliaLang/juliaup/pull/1002. Pixi will pull in the latest juliaup with this patch.